packages/cli/e2e/__tests__/deploy.spec.ts

```diff
@@ -20,25 +20,32 @@ async function cleanupProjects (projectLogicalId?: string) {
     },
   })
   const projectsApi = new Projects(api)
+  if (projectLogicalId) {
+    await projectsApi.deleteProject(projectLogicalId)
+    return
+  }
   const { data: projects } = await projectsApi.getAll()
   for (const project of projects) {
-    const matchesLogicalId = project.logicalId === projectLogicalId
     // Also delete any old projects that may have been missed in previous e2e tests
     const leftoverE2eProject = project.name.startsWith('e2e-test-deploy-project-') &&
-      DateTime.fromISO(project.created_at) < DateTime.now().minus(Duration.fromObject({ minutes: 10 }))
-    if (matchesLogicalId || leftoverE2eProject) {
+      DateTime.fromISO(project.created_at) < DateTime.now().minus(Duration.fromObject({ minutes: 20 }))
```

Comment:
I wonder if there's a chance that `created_at` isn't consistent with the time in the GH action. That could delete some projects while they're in-use in other tests.

Reply:
Valid concern but it is just the test projects. We can detect that with test failures later on